crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -1164,6 +1167,12 @@ impl PlotImage {
         self
     }
 
+    /// Rotate the image counter-clockwise around its center.
```

Comment:
egui uses clock-wise as the standard rotation direction. See for instance `Rot2::from_angle`, `Vec2::angled`, `TextShape::angle`, …

Reply:
Also, it is worth noting that this is in radians.

Reply:
Indeed, but the plot coordinates are already different from the screen coordinates. As the y-axis points up in those coordinates, it would make more sense for the rotation to be counter-clockwise according the the right-hand rule.

Reply:
Good point, I didn't think of that!